langchain/src/stores/tests/convex.int.test.ts

```diff
@@ -0,0 +1,56 @@
+/* eslint-disable no-process-env */
+
+import { ConvexHttpClient } from "convex/browser";
+
+import { expect, test } from "@jest/globals";
+// eslint-disable-next-line import/no-relative-packages
+import { api } from "./convex/convex/_generated/api.js";
```

Comment:
OOC a dev would need to know this path to use Convex?

Reply:
All TypeScript clients use a path like this. See Step 9 in this quick start for an example: https://docs.convex.dev/quickstart/react

But a vanilla string could be used instead, without typechecking that the endpoint exists.